vnet/lib/vnet/core/dp_info.rb

```diff
@@ -171,6 +181,21 @@ def terminate_managers(timeout = 10.0)
       }
     end
 
+    def bootstrap_managers
+      BOOTSTRAP_MANAGER_NAMES.map { |name| __send__("#{name}_manager") }
```

Comment:
It copied the other 'managers' method, which uses '__send__'.

Likely leftover from when dp_info was an actor.